lv_conf_template.h

```diff
@@ -695,6 +695,17 @@
     #endif // LV_IME_PINYIN_USE_K9_MODE
 #endif
 
+/*1: Enable Pinyin input method*/
+/*Requires: lv_table*/
```

Comment:
If it requires `lv_table`, shouldn't it be conditioned (`#ifdef`) in `LV_USE_TABLE`?

Reply:
This is a good suggestion. Here I follow the previous format. If it needs to be modified, I think it should be modified together after file explorer merge. @kisvegabor

Reply:
It really looks like as @100ask did with other widgets too. We can reconsider it, but it should be an other PR.